domain/src/main/java/org/oppia/android/domain/profile/ProfileManagementController.kt

```diff
@@ -39,7 +39,7 @@ import java.io.FileOutputStream
 import javax.inject.Inject
 import javax.inject.Singleton
 
-private const val DEFAULT_LOGGED_OUT_INTERNAL_PROFILE_ID = -1
+private val DEFAULT_LOGGED_OUT_PROFILE_ID = ProfileId.newBuilder().setLoggedOut(true).build()
```

Comment:
Semantically more correct now:
```suggestion
private val LOGGED_OUT_PROFILE_ID = ProfileId.newBuilder().setLoggedOut(true).build()
```

Reply:
Done.